app/pulp/app/viewsets/repository.py

```diff
@@ -41,3 +44,34 @@ def content(self, request, name):
         page = paginator.paginate_queryset(repo.content, request)
         serializer = ContentSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
+
+    @decorators.detail_route()
+    def importers(self, request, name):
+        # Creates a repositories/<repo>/importers/ endpoint.
+        # This will link to all importers that are associated within the repository.
+        repo = self.get_object()
+        importers = Importer.objects.filter(repository__name=repo.name)
+        paginator = UUIDPagination()
+        page = paginator.paginate_queryset(importers, request)
+        serializer = ImporterSerializer(page, many=True, context={'request': request})
+        return paginator.get_paginated_response(serializer.data)
```

Comment:
This is so similar to the content detail route that it would be useful to have a sort of factory function to generalize the combination of a queryset, paginator class, and serializer class into a paginated response.